pkg/bindings/volumes/volumes.go

```diff
@@ -139,3 +144,59 @@ func Exists(ctx context.Context, nameOrID string, options *ExistsOptions) (bool,
 
 	return response.IsSuccess(), nil
 }
+
+// Export exports a volume to the given path
+func Export(ctx context.Context, nameOrID string, options entities.VolumeExportOptions) error {
```

Comment:
I would expect the bindings to just return the stream so the caller can write or do what they want, otherwise you force all bindings users to a temporary file if they need to inspect the stream for whatever reasons

Reply:
Oh and also bindings options don't use entities structs, they should use specific bindings types